### PR TITLE
Staging v1.11.0

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
Release activities for version 1.11.0 GA in issue #1621

### CODE REVIEW

**Review by richp405 — APPROVED** [review]

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -6,84 +6,92 @@
...
 
 Bugfixes
 --------
 
-- ``zos_copy`` - when creating the destination data set, the module would unnecessarily check if a data set is locked by another process. The module no longer performs this check when it creates the data set.
+- ``zos_copy``
+
+   - Fixes the issue that prevents the module from automatically computing member names when copying a file into a PDS/E. The module now computes the member name when copying into a PDS/E.
+   - Fixes an issue that would perform an unnecessary check if a destination data set is locked for data sets the module created. The module now only performs this check for on destinations that are present.
+
```
Comment (ketankelkar) [inline docs/source/release_notes.rst:91]: > ...now only performs this check for on destinations that are present.

should remove the "on" (or remove from):
> ..now only performs this check for ~on~ destinations that are present.

File: docs/source/modules/zos_lineinfile.rst [hunk 2]
```diff
@@ -33,7 +33,7 @@ src
 
   The USS file must be an absolute pathname.
 
-  Generation data set (GDS) relative name of generation already created. ``e.g. SOME.CREATION(-1.)``
+  Generation data set (GDS) relative name of generation already created. e.g. *SOME.CREATION(-1*).
```
Comment (ketankelkar) [inline docs/source/modules/zos_lineinfile.rst:36]: it seems to me like the '*' should be placed outside the close paren ')'
> e.g. *SOME.CREATION(-1*).

File: docs/source/modules/zos_blockinfile.rst [hunk 3]
```diff
@@ -33,7 +33,7 @@ src
 
   The USS file must be an absolute pathname.
 
-  Generation data set (GDS) relative name of generation already created.  ``e.g. SOME.CREATION(-1``.)
+  Generation data set (GDS) relative name of generation already created. e.g. *SOME.CREATION(-1*).
```
Comment (ketankelkar) [inline docs/source/modules/zos_blockinfile.rst:36]: similar comment as before, seems like the '*' should be after the close paren ')'

**Review by ketankelkar — CHANGES_REQUESTED** [review]
A few minor things in doc which may or may not be addressable. Other than that, this merge looks good to me

File: docs/source/modules/zos_lineinfile.rst (hunk 2 above)
Comment (ddimatos) [reply docs/source/modules/zos_lineinfile.rst:36]: Does seem like that but the doc decided to generate it this way and I reviewed it, see below 
image

File: docs/source/modules/zos_blockinfile.rst (hunk 3 above)
Comment (ddimatos) [reply docs/source/modules/zos_blockinfile.rst:36]: Same as above comment only different module 
image

File: docs/source/release_notes.rst (hunk 1 above)
Comment (ddimatos) [reply docs/source/release_notes.rst:91]: Done

File: docs/source/release_notes.rst [hunk 4]
```diff
@@ -6,84 +6,92 @@
...
+- ``zos_blockinfile``
+
+   - Added support for GDG and GDS relative name notation to specify a data set. And backup in new generations.
+   - Added support for data set names with special characters ($, /#, /- and @).
 
 - ``zos_copy``
 
-   - a regression in version 1.4.0 made the module stop automatically computing member names when copying a single file into a PDS/E. Fix now lets a user copy a single file into a PDS/E without adding a member in the dest option.
-   - module would use opercmd to check if a non existent destination data set is locked. Fix now only checks if the destination is already present.
+   - Added support for copying from and too generation data sets (GDS) and generation data groups (GDG) including using a GDS for backup.
```
[the remark this answers is not in the document]
Comment (ddimatos) [reply docs/source/release_notes.rst]: done

**Review by ketankelkar — APPROVED** [review]
LGTM

**Pull request merged**